plugin-server/src/cdp/consumers/cdp-behavioural-events.consumer.ts

```diff
@@ -16,6 +18,7 @@ import { CdpConsumerBase } from './cdp-base.consumer'
 export type BehavioralEvent = {
     teamId: number
     filterGlobals: HogFunctionFilterGlobals
+    personId?: string
```

Comment:
I'm pretty sure personId would _always_ be set (if it isn't it should be dropped at the parsing stage)